### ISSUE TITLE
Support for multiple displays

### ISSUE THREAD

**Issue by mkuron** [qvacua/qdesktop#5]
When multiple displays are connected to the Mac, Qdesktop always runs on the primary screen. It should be capable of spanning a website across multiple desktops or displaying a different website on each desktop.

**Comment by qvacua** [comment]
Yes, that would be definitely useful. The problem is to find time to implement it.... :cry:
